### DISCUSSION TITLE
[Feat]支持多token账号切换

### DISCUSSION THREAD

**Discussion by MrTreasure** [immersive-translate/immersive-translate#1717] [Ideas]
### 新功能 | New Feature

1.支持配置多套token进行切换

### 为什么您想要添加这些功能 | Why you want to add these features

1.公司内网提供独立的 gpt 能够直接翻译，但是个人电脑chrome账户是相同的。个人电脑没接入内网，拥有自己的gpt token 希望能支持多个配置，方便切换

### 提议 | Proposal

在token配置选项中支持多个配置项

**Comment by theowenyoung** [comment]
接下来会支持添加 OpenAI 适配层
